app/graphql/resolvers/submissions_resolver.rb

```diff
@@ -54,6 +58,10 @@ def user_ids
     @arguments.fetch(:user_id, [])
   end
 
+  def filter_by_category
+    @arguments.fetch(:filter_by_category, [])
```

Comment:
I'm wondering if there's a bug here when one does not supply the `filter_by_category` argument. The purpose of the `compact` above is to remove conditions that don't apply, but an empty array is not affected:

```rb
irb(main):001:0> { category: [] }.compact
=> {:category=>[]}
```

So, I'm concerned that if the `fetch` here returned an empty array that we'd inadvertently filter to only submissions with NO CATEGORY! Does that make sense? Happy to hop on a call to kick the tires and maybe you've already thought of this case, but reading through this my spidey sense was triggered, haha! 🕷️

Reply:
I tried the query in all kinds of ways and didn't see a problem, though I could also be misundestanding; here's a couple screenshots: 

<img width="961" alt="Screen Shot 2020-07-31 at 10 18 55 AM" src="https://user-images.githubusercontent.com/236943/89060151-5f9c2a80-d317-11ea-81b5-cb06e22a025c.png">

<img width="994" alt="Screen Shot 2020-07-31 at 10 20 22 AM" src="https://user-images.githubusercontent.com/236943/89060245-8d816f00-d317-11ea-9b3e-3c259b21e7b3.png">

![Uploading Screen Shot 2020-07-31 at 10.20.53 AM.png…]()

Reply:
Thanks for humoring me!! 🤗